docs/content.zh/docs/connectors/datastream/dynamodb.md

```diff
@@ -0,0 +1,171 @@
+---
+title: DynamoDB
+weight: 5
+type: docs
+aliases:
+- /zh/dev/connectors/dynamodb.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Amazon DynamoDB Sink
+
+The DynamoDB sink writes to [Amazon DynamoDB](https://aws.amazon.com/dynamodb) using the [AWS v2 SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html). Follow the instructions from the [Amazon DynamoDB Developer Guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-1.html)
+to setup a table.
+
+To use the connector, add the following Maven dependency to your project:
+
+{{< connector_artifact flink-connector-dynamodb 2.0.0 >}}
```

Comment:
> use the connector_artifact shortcode and supply the version of the connector without the Flink version suffix (1.0.0).
The Flink docs then integrate the docs of the connector version that is compatible with Flink, and the connector_artifact shortcode injects the corresponding version suffix.

@zentol  does this look right? When I run locally using tip of Flink master I do not get any output. I cannot see where `connector_artifact` is implemented, I can see the [readme reference](https://github.com/apache/flink/blob/master/docs/README.md#L149) only. Looks like the build of elasticsearch docs is using `{{< artifact flink-connector-elasticsearch6 >}}` still, since it is pulling the 3.0.0 branch.

Reply:
Flink implementation:
https://github.com/apache/flink/blob/release-1.16/docs/layouts/shortcodes/connector_artifact.html
ES usage: https://github.com/apache/flink-connector-elasticsearch/commit/b32f832f19ee52c43cddc0f32e835ef9c208a770

However, it does indeed not show correctly in the live docs for some reason. Will look into it...

Reply:
Ah of course it doesn't show it; the Flink docs use the v3.0.0 tag, and that just doesn't include the commit above.